### PR TITLE
pull request for adding j~n gunintam

### PR DESCRIPTION
**Pull request by tuxnani**
pull request for adding j~n gunintam

### CODE REVIEW

File: test/jquery.ime.test.fixtures.js [hunk 1]
```diff
@@ -2333,7 +2333,9 @@ var palochkaVariants = {
 		description: 'Telugu Transliteration test',
 		tests: [
 			{ input: 'c', output: 'చ్', description: 'Telugu c' },
-			{ input: 'ch', output: 'చ్', description: 'Telugu ch' }
+			{ input: 'ch', output: 'చ్', description: 'Telugu ch' },
+			{ input: 'j~nu', output: 'జ్ఞు', description: 'Telugu j~nu' },
+			{ input: 'ఞా', output: '~naa', description: 'Telugu ~naa' }
```
Comment (santhoshtr) [inline test/jquery.ime.test.fixtures.js]: This does not look good. Input in telugu and output in english?

**Comment by tuxnani** [comment]
Changes done in my codebase, how to reflect it here?

**Comment by arjunaraoc** [comment]
@tuxnani ,  can you address the conflicting files warning at the earliest  to take this forward

**Comment by tuxnani** [comment]
Corrected as required.

**Comment by arjunaraoc** [comment]
@tuxnani Thanks.  I did not see a fresh commit after fix for conflicting files. Also I did not understand why the issue was closed.  Can you clarify?

**Pull request closed without merging**